content/en/docs/concepts/architecture/cgroups.md

```diff
@@ -103,6 +103,7 @@ updated to newer versions that support cgroup v2. For example:
 * If you run [cAdvisor](https://github.com/google/cadvisor) as a stand-alone
  DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
 * If you use JDK, prefer to use JDK 11.0.16 and later or JDK 15 and later, which [fully support cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305).
+* If you use uber-go/automaxprocs, update it to v1.5.1 or later.
```

Comment:
is this package relevant to cgroup v2?

Reply:
@bobbypage do you have an insight here?

Reply:
@tengqm like JDK, automaxprocs depends on cgroupfs to support container awareness.

Without it, an application is likely to hit the [cgroup throttling issue](https://danluu.com/cgroup-throttling) issue. I think it's a block issue of migrating to v2. AFAIK, it would lots of companies that depend on Go.

Reply:
@wenjianhn Thanks for the clarification.

```suggestion
* If you are using [uber-go/automaxprocs](https://github.com/uber-go/automaxprocs) package, make sure
  the version you use is v1.5.1 or higher.
```

Reply:
@tengqm done